### ISSUE TITLE
Your requirements could not be resolved

### ISSUE THREAD

**Issue by Enferum** [wdev-rs/laravel-datagrid#8]
Problem:
```
user@user:~$ composer require wdev-rs/laravel-datagrid
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires wdev-rs/laravel-datagrid ^0.3.0 -> satisfiable by wdev-rs/laravel-datagrid[0.3.0].
    - wdev-rs/laravel-datagrid 0.3.0 requires illuminate/support ^6.0|^7.0|^8.0 -> found illuminate/support[v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.


Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

Composer.json:
```
{
    "require": {
        "php": "^8.0.2",
        "guzzlehttp/guzzle": "^7.2",
        "jeroennoten/laravel-adminlte": "^3.8",
        "laravel/framework": "^9.2",
        "laravel/sanctum": "^2.14.1",
        "laravel/tinker": "^2.7",
        "laravel/ui": "^4.0"
    },
    "require-dev": {
        "fakerphp/faker": "^1.9.1",
        "laravel/sail": "^1.0.1",
        "mockery/mockery": "^1.4.4",
        "nunomaduro/collision": "^6.1",
        "phpunit/phpunit": "^9.5.10",
        "spatie/laravel-ignition": "^1.0"
    },
    "autoload": {
        "psr-4": {
            "App\\": "app/",
            "Database\\Factories\\": "database/factories/",
            "Database\\Seeders\\": "database/seeders/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "scripts": {
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi"
        ],
        "post-install-cmd": [
            "composer licenses > LICENSE-DEPENDENCIES.md"
        ],
        "post-update-cmd": [
            "@php artisan vendor:publish --tag=laravel-assets --ansi --force",
            "composer licenses > LICENSE-DEPENDENCIES.md"
        ],
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate --ansi"
        ]
    },
    "extra": {
        "laravel": {
            "dont-discover": []
        }
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": "dist",
        "sort-packages": true
    },
    "minimum-stability": "dev",
    "prefer-stable": true
}
```

**Does this package support Laravel 9?**

**Comment by daniel-werner** [comment]
@Enferum Thanks for reporting, I'll check the L9 compatibility, and will release a new version soon if necessary.

**Comment by daniel-werner** [comment]
Hey @Enferum, the newly released version 0.4.0 supports Laravel 9, and also includes some minor improvements in functionality as well.
